test/local/account_routes.js

```diff
@@ -959,64 +959,15 @@ test('/account/login', function (t) {
         t.deepEqual(args[1], 'account.confirmed', 'second argument was event name')
         t.equal(args[2], mockRequest.payload.metricsContext, 'third argument was metrics context')
         t.deepEqual(mockMetricsContext.stash.args[2][1], 'account.keyfetch', 'third call was for account.keyfetch')
-      }).then(function () {
-        mockMailer.sendVerifyLoginEmail.reset()
-        mockDB.createSessionToken.reset()
-        mockMetricsContext.stash.reset()
-      })
-    })
 
-    t.test('location data is present in sign-in confirmation email', function (t) {
-      return runTest(route, mockRequest, function (response) {
         t.equal(mockMailer.sendVerifyLoginEmail.callCount, 1, 'mailer.sendVerifyLoginEmail was called')
         t.equal(mockMailer.sendVerifyLoginEmail.getCall(0).args[2].location.city, 'Mountain View')
         t.equal(mockMailer.sendVerifyLoginEmail.getCall(0).args[2].location.country, 'United States')
         t.equal(mockMailer.sendVerifyLoginEmail.getCall(0).args[2].timeZone, 'America/Los_Angeles')
       }).then(function () {
         mockMailer.sendVerifyLoginEmail.reset()
         mockDB.createSessionToken.reset()
-      })
-    })
-
-    t.test('on for sample', function (t) {
-      // Force uid to '01...'
-      uid.fill(0, 0, 1)
-      uid.fill(1, 1, 2)
```

Comment:
This test and the next one were broken by the new cohort sampling algorithm. The assertions didn't appear to be covering anything not already covered elsewhere, so all they were doing was duplicating the coverage that is now in `test/local/features.js`. Hence I removed them, in an effort to make this overgrown test file a little bit more manageable.